aws_lambda_powertools/utilities/feature_flags/feature_flags.py

```diff
@@ -87,6 +90,10 @@ def _evaluate_conditions(
             cond_action = condition.get(schema.CONDITION_ACTION, "")
             cond_value = condition.get(schema.CONDITION_VALUE)
 
+            # rule based actions have no user context. the context is the condition key
+            if cond_action == schema.RuleAction.TIME_RANGE.value or schema.RuleAction.TIME_SELECTED_DAYS:
+                context_value = condition.get(schema.CONDITION_KEY)
+
```

Comment:
```suggestion
            # rule based actions have no user context. the context is the condition key
            if cond_action == schema.RuleAction.TIME_RANGE.value or schema.RuleAction.TIME_SELECTED_DAYS:
                context_value = condition.get(schema.CONDITION_KEY)
                
                if cond_action == schema.RuleAction.TIME_SELECTED_DAYS:
	                if not isinstance(cond_value,list):
	                    cond_value = [cond_value]
```

With user experience in mind, a value for this condition can be a simple string or a list, so we convert it to a list if it's a string. 
That makes sense?

Reply:
Not sure I agree with this one. I see this action as "IN" so it should be a list.
If you open it to duality of values, it's more confusing.
It also makes inputs such as 'MONDAY, TUESDAY' break the code since you will turn it into a ['MONDAY, TUESDAY' ] which is not correct. I think having it as a list, and validating it as a list will keep it simple.